src/workerd/api/node/node.h

```diff
@@ -29,6 +30,53 @@ class CompatibilityFlags : public jsg::Object {
   }
 };
 
+class Process : public jsg::Object {
+public:
+  v8::Local<v8::Object> getEnv(jsg::Lock& js, bool rootContext = true) {
```

Comment:
Nitpick, but there's a lot of code in these inline functions, which will end up being compiled as part of workerd-api.c++ (and the corresponding internal file) because that's the place that actually calls it, making that file take even longer to compile. Moving the code out-of-line helps avoid that.

Reply:
Fair point :-)